onnx_tf/frontend.py

```diff
@@ -217,14 +264,23 @@ def handle_random_standard_normal(cls, node, consts):
             seed=node.attr["seed"],
             mean=0.0,
             scale=1.0,
-            shape=node.attr["_output_shapes"][0])
+            shape=shape)
 
   @classmethod
-  def handle_random_uniform(cls, node, consts):
+  def handle_random_uniform(cls, node, consts, input_node):
     """ Tensorflow does not have a generic random_uniform op.
         The generic random_uniform op is translated into a scaled
         and offsetted random standard uniform op.
     """
+    shape = node.attr["_output_shapes"][0]
+    # Arrange shape to the according data format 
+    if cls._channel_last:
+      rank = len(shape)
+      if rank >= 3:
+        old_shape = shape[:]
+        for i, s in enumerate(shape):
+          j = i % (rank-1) + 1 if i != 0 else 0
```

Comment:
I have seen this pattern quite a bit. Can you use a helper function to perform this NHWC->NCHW transformation? This could increase readability quite a bit.

Reply:
Of course, I will do the helper function as soon as possible.